src/main/features/claim/form/models/claimAmountRow.ts

```diff
@@ -38,13 +38,27 @@ export class ClaimAmountRow {
     return new ClaimAmountRow(undefined, undefined)
   }
 
+  static translateToNumber (input?: any): any {
+    const numberValue = _.toNumber(input)
+    const isExponential = numberValue.toString().match(new RegExp('\\d*[Ee][+-]?\\d*?'))
+
+    if (!isNaN(numberValue) && !isExponential) {
+      return numberValue
+    } else {
+      return input
```

Comment:
Method should return undefined. A `toNumberOrUndefined` does it right.

Reply:
method returns a number or string (original value) so that user input is not lost in conversion as the requirement